src/ticks.jl

```diff
@@ -114,17 +114,15 @@ function optimize_ticks_typed{T}(x_min::T, x_max::T, extend_ticks,
                     g = 0 < k < 2k_ideal ? 1 - abs(k - k_ideal) / k_ideal : 0.0
 
                     # coverage
-                    c = 1.5 * xspan/span
+                    c = 1.5*xspan/span
 
                     score = granularity_weight * g +
                             simplicity_weight * s +
                             coverage_weight * c +
                             niceness_weight * qscore
 
-                    # strict limits on coverage
-                    if strict_span && span > xspan
-                        score -= 10000
```

Comment:
Adding
```
if span >= α*xspan
    score -= 1000
end
```
here should be sufficient for your case (depending on α ... 4 works, 4.5 is already too big) and would not affect the `strict_span == false` case